app/models/external_training.rb

```diff
@@ -15,14 +15,16 @@ class ExternalTraining < ActiveRecord::Base
 
   validates_date :finish_at, on_or_after: :start_at, allow_blank: true
 
+  validates_date :finish_at, on_or_before: lambda { Date.current }
+
   scope :list, -> { order(created_at: :desc) }
 
   after_destroy :revoke_qualifications
   after_save :issue_qualifications
   after_save :create_trainings_for_other_people
 
   def self.between(start_date, end_date)
-    where('start_at <= :end_date AND finish_at >= :start_date ',
+    where('start_at <= :end_date AND finish_at >= :start_date',
           start_date: start_date, end_date: end_date).distinct
```

Comment:
This can be simplified:
```suggestion
    where(start_at: ..end_date, finish_at: start_date..).distinct
```